InternetRadioDevice/InternetRadio/Properties/AssemblyInfo.cs

```diff
@@ -6,11 +6,11 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("InternetRadio")]
-[assembly: AssemblyDescription("Internet Radio Headless Application for Microsoft IoT Core")]
+[assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
+[assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("InternetRadio")]
-[assembly: AssemblyCopyright("Copyright Microsoft ©  2015")]
+[assembly: AssemblyCopyright("Copyright ©  2015")]
```

Comment:
Is this correct? Shouldn't copyright still be attributed to Microsoft? Not sure if it matters, since LCA has only ever required us to declare copyright in a license file